gematria/datasets/find_accessed_addrs_test.cc

```diff
@@ -107,9 +108,10 @@ class FindAccessedAddrsTest : public testing::Test {
 };
 
 TEST_F(FindAccessedAddrsTest, BasicMov) {
-  EXPECT_THAT(FindAccessedAddrsAsm("mov [0x10000], eax"),
-              IsOkAndHolds(Field(&BlockAnnotations::accessed_blocks,
-                                 ElementsAre(0x10000))));
+  EXPECT_THAT(
+      FindAccessedAddrsAsm("mov [0x10000], eax"),
+      IsOkAndHolds(testing::Property(&ExecutionAnnotations::accessed_blocks,
+                                     ElementsAre(0x10000))));
```

Comment:
With protos in place, we can use the [`EqualsProto()` matcher](https://github.com/google/gematria/blob/129049615bee55ea38f5ebf574b0529bffd33815/gematria/testing/matchers.h#L131).

```cc
  EXPECT_THAT(
      FindAccessedAddrsAsm("mov [0x10000], eax"),
      IsOkAndHolds(Partially(EqualsProto("accessed_blocks: 0x10000"))));
```
though the full proto would not be much longer.

Same for the other tests.